source/06-logging/01-log-file-analysis.rst

```diff
@@ -5,10 +5,16 @@
 Log File Analysis
 -----------------
 
-The main advantages of log file analysis over page tagging are:
+Log files are text files representing individual HTTP requests, including the user host name or IP address, the date and time of the request, the requested file name, the HTTP response status and size, the referring URL and the browser information. HTTP requests are what happen when a user tries to access a web page, while an HTTP response is the delivery of that page to the user’s browser.
 
-* Web servers normally produce log files, so the raw data are already available. No changes to the website are required.
-* The data is on the organization’s own servers and is in a standard, rather than a proprietary, format. This makes it easy for an organization to switch programs later, use several different programs, and analyse historical data with a new program.
-* Log files contain information on visits from search engine spiders. Although these MUST NOT be reported as part of user activity, it is useful information for search engine optimization.
+Most web servers produce log files by default in a pre-defined format which may differ by server type. The log file data needed for a COUNTER implementation should be available without changes to the platform.
+
+As log file data is held on servers in a standard format, report providers can use a variety of analytics programmes and receive consistent results over time. Log files are also independent of users’ web browsers, meaning report providers can reliably track all activity for the purposes of COUNTER reporting.
+
+Report providers wishing to learn more about log files should read the Amazon Web Service documentation (https://docs.aws.amazon.com/).
```

Comment:
I noticed that you usually put URLs in round brackets which differs from what we've done so far. This should be made consistent, probably in the clean up pull request.